<?xml version="1.0"?>

<launch>

	<node name="navigToPoint" pkg="slam_project_buggy" type="navigToPoint.py"/>

	<!-- <node name="createPath" pkg="slam_project_buggy" type="createPath.py"/> -->



</launch>
